plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerMetadata.java

```diff
@@ -457,13 +467,30 @@ else if (metadata.getStatisticType().equals(ColumnStatisticType.MAX_VALUE)) {
         if (views.containsKey(viewName)) {
             throw new TrinoException(ALREADY_EXISTS, "View already exists: " + viewName);
         }
-        String whereClause = minimums.isEmpty() ? "" : "WHERE " + minimums.entrySet().stream()
-                .map(entry -> "%s BETWEEN %s AND %s".formatted(
-                        entry.getKey(),
-                        formatLiteral(types.get(entry.getKey()), entry.getValue()),
-                        formatLiteral(types.get(entry.getKey()), maximums.get(entry.getKey()))))
-                .collect(joining(" AND "));
-        String viewSql = "SELECT * FROM %s.%s %s".formatted(
+        List<String> viewColumns = columns.stream()
+                .filter(column -> !ROW_ID_COLUMN_NAME.equals(column.name()))
+                .map(column -> {
+                    String name = column.name();
+                    Type type = types.get(name);
+                    if (!type.equals(BIGINT) || !minimums.containsKey(name)) {
+                        return quoted(name);
+                    }
+                    if ((double) distinctValues.get(name) / rowCount.get() < 1) {
```

Comment:
I do not get why are you using `< 1` here.
Should it be rather something like `abs(1.0 - (double) distinctValues.get(name) / rowCount.get()) < some_epsilon`?

Reply:
Must have been late when I wrote this, this shows I need better test cases.

Reply:
This is actually correct - I just changed 1 to 0.98, I should definitely make it a constant, but I was wondering if it also should be configurable.